internal/services/kusto/kusto_cluster_resource_test.go

```diff
@@ -400,9 +400,10 @@ resource "azurerm_kusto_cluster" "test" {
   location                      = azurerm_resource_group.test.location
   resource_group_name           = azurerm_resource_group.test.name
   public_network_access_enabled = false
+  public_ip_type = "DualStack"
   sku {
-    name     = "Dev(No SLA)_Standard_D11_v2"
-    capacity = 1
+name     = "Standard_D13_v2"
```

Comment:
You can run `make terrafmt` to format these terraform blocks. You might need to run `make tools` to download the necessary tools.

Reply:
Update this manually for now